docs/io/configuration/components/models/converters/arepo_to_tardis.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<p>Could you hyperlink the section on the CSVY model?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>